projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -2699,6 +2699,18 @@ t_server
    SERVER hostname = variable_hostname NEWLINE
 ;
 
+t_address
+:
+   (
+      ADDRESS ((IPV4 IP_ADDRESS) | (IPV6 IPV6_ADDRESS)) NEWLINE
+   )
+;
+
+t_key
```

Comment:
Then the same should apply to F_Digit fragment as well; change followed the existing code base.